tamr_unify_client/models/dataset/resource.py

```diff
@@ -104,27 +104,37 @@ def refresh(self, **options):
         op = Operation.from_json(self.client, op_json)
         return op.apply_options(**options)
 
-    def profile(self, **options):
-        """Returns up to date profile information for a dataset, re-profiling if not up to date.
+    def profile(self):
+        """Returns profile information for a dataset.
+
+        If profile information has not been generated, call create_profile() first.
+        If profile information is out-of-date, call refresh() on the returned
+        object to bring it up-to-date.
```

Comment:
According to the ticket shouldn't we just be fetching the dataset in its current state as opposed to calling a refresh() on it?